pytorch_lightning/loops/base.py

```diff
@@ -46,8 +46,6 @@ class Loop(ABC):
     """
 
     def __init__(self) -> None:
-        # TODO: replace by progress tracking
-        self.iteration_count: int = 0
```

Comment:
can we have some dummy progress here then? So that a user that quickly experiments with it can still have some kind of progress tracking built-in?

Reply:
Answering below